contrib/segmentio/kafka.go.v0/option.go

```diff
@@ -75,3 +79,17 @@ func WithAnalyticsRate(rate float64) Option {
 		}
 	}
 }
+
+// WithDataStreams enables the Data Streams monitoring product features: https://www.datadoghq.com/product/data-streams-monitoring/
+func WithDataStreams() Option {
+	return func(cfg *config) {
+		cfg.dataStreamsEnabled = true
+	}
+}
+
+// WithGroupID tags the produced data streams metrics with the given groupID (aka consumer group)
+func WithGroupID(groupID string) Option {
+	return func(cfg *config) {
+		cfg.groupID = groupID
+	}
+}
```

Comment:
This is what I had at first before changing it following https://github.com/DataDog/dd-trace-go/pull/2625#discussion_r1537820468

Reply:
I replied here: https://github.com/DataDog/dd-trace-go/pull/2625#discussion_r1547488982

Sorry for the confusion! @adrien-f but TLDR; the reason we have that option in Sarama is because we don't have a way to access the groupID there, but in this contrib and the confluent kafka one we do have the option to access it from the config, so this would make this `WithGroupID` option unnecessary.